docs/evm/guides/wagmi.md

```diff
@@ -1,14 +1,37 @@
 ---
-title: Wagmi
+title: Viem & Wagmi
 description: "Using Wagmi to interact with Solidity contract to EVM on Flow."
-sidebar_label: Wagmi
+sidebar_label: Viem & Wagmi
 sidebar_position: 4
 ---
 
 :::info
 Make sure to use `viem` version `2.9.6` or greater. This version contains flow EVM networks
 :::
 
+
+# Using viem
```

Comment:
```suggestion
# Using Viem
```

Reply:
Would be nice to have a brief description of what viem is: TypeScript Interface for Ethereum
Same goes for Wagmi. Could be handled in a simple intro paragraph or something